data/map planets.txt

```diff
@@ -810,9 +810,9 @@ planet Darkstone
 	spaceport `Once every few months, one pirate band or another will land here demanding tribute in the form of food. The locals respond by just keeping a big enough supply on hand to satisfy them; any sort of planetary defenses would cost far more than the price of the amount of food that an average fleet of warships can carry. But once they recognize that you are here to trade and not to steal, they greet you quite enthusiastically.`
 	outfitter "Ammo South"
 	security 0.1
-	tribute 200
-		threshold 1500
-		fleet "Small Militia" 18
+	tribute 150
+		threshold 800
+		fleet "Small Southern Pirates" 8
```

Comment:
I don't think it makes sense for this tribute fleet to change. The Free Worlds ultimately does get a grip on piracy in the region, yet this planet would still have a pirate defense fleet.
```suggestion
	tribute 200
		threshold 1500
		fleet "Small Militia" 18
```

Reply:
mb it was changed so the FW wouldnt be mad at you for this one world
that said it could be a better idea to ask to tribute a pirate planet with an already weak defense force?